gitcoins-cordapp/workflows/src/main/kotlin/com/gitcoins/flows/PushEventFlow.kt

```diff
@@ -0,0 +1,47 @@
+package com.gitcoins.flows
+
+import co.paralleluniverse.fibers.Suspendable
+import com.gitcoins.schema.GitUserMappingSchemaV1
+import com.r3.corda.sdk.token.contracts.utilities.of
+import com.r3.corda.sdk.token.workflow.flows.IssueToken
+import com.gitcoins.states.GitToken
+import net.corda.core.crypto.Crypto
+import net.corda.core.flows.*
+import net.corda.core.identity.AnonymousParty
+import net.corda.core.identity.CordaX500Name
+import net.corda.core.identity.Party
+import net.corda.core.transactions.SignedTransaction
+import net.corda.core.utilities.base58ToByteArray
+import java.security.PublicKey
+
+/**
+ * Flow that delegates the issuing of a [GitToken] to the [IssueToken] subflow. This flow is triggered by a GitHub push
+ * event that is linked to a github webhook.
+ */
+@StartableByRPC
+class PushEventFlow(private val gitUserName: String) : FlowLogic<SignedTransaction>() {
```

Comment:
This flow is exactly the same as the `PullRequestReviewEventFlow`

Reply:
The idea is that eventually there will be logic to evaluate the 'value' of a push event and pr review so these will need to be separated.